src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -129,7 +134,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $input->getOption('router'), $env, null)) {
+        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $router, null)) {
```

Comment:
Fair enough, I was just wary of changing more than was necessary - I'm pretty new to GitHub so wasn't sure if I would be committing a faux pas!
All done in 139191f4623f5a4ef83ed26bd2eedcdbdd0992ad.